mlflow/langchain/runnables.py

```diff
@@ -74,6 +74,10 @@ def _load_model_from_config(path, model_config):
         return load_llm(config_path)
     elif _type in custom_type_to_loader_dict():
         return custom_type_to_loader_dict()[_type](config)
+    elif _type == "openai-chat":
+        from langchain_openai import ChatOpenAI
+        config.pop("_type")
+        return ChatOpenAI(**config)
```

Comment:
Only for test: This change is for easier testing locally with `ChatOpenAI` model.

Reply:
what would happen if we didn't have this elif branch?

Reply:
Loading failed.

Reply:
can you elaborate?

Reply:
Actually can we add this to custom_type_to_loader_dict as part of our fix?
@harupy Langchain doesn't put ChatOpenAI as part of their llms_get_type_to_cls_dict mapping, so we fails to load it here

Reply:
yes. but I think we should update langchain code `custom_type_to_loader_dict` instead.